### DISCUSSION TITLE
Firefly III DB : unauthenticated host : ’172.31.0.4‘(this connection closed normally without authentication)

### DISCUSSION THREAD

**Discussion by shibinbin8** [firefly-iii/firefly-iii#9531] [Firefly III]
This is an error shows in FireflyIII DB , I used FireflyIII 1 month until one day it shows this , can some one help me, now I can't open web page
![db error](https://github.com/user-attachments/assets/5ed5f52e-2ca8-4aa2-97ae-df8915b01ffa)
![image](https://github.com/user-attachments/assets/ce92a39d-d307-49b2-80c1-91bf2185040c)

**Comment by JC5** [comment]
> this connection closed normally without authentication

This is not an issue in itself, it is caused by a script that checks if the database is up.

What is happening otherwise? I see no errors.

**Reply by shibinbin8** [reply]
I can't open webpage

**Reply by shibinbin8** [reply]
unauthenticated host : ’172.31.0.4‘(this connection closed normally without authentication)


The ip:172.31.0.4 is ap's address

**Reply by JC5** [reply]
Could you be a little more specific please.

**Comment by shibinbin8** [comment]
1. webpage:
![image](https://github.com/user-attachments/assets/949937d3-f455-431e-a5f2-1608e5475ef9)

2. services:
  app:
    image: fireflyiii/core:latest
    hostname: app
    container_name: firefly_iii_core
    restart: always
    volumes:
      - firefly_iii_upload:/var/www/html/storage/upload
    env_file: .env
    networks:
      - firefly_iii
    ports:
      - 7787:8080
    depends_on:
      - db
  db:
    image: mariadb:lts
    hostname: db
    container_name: firefly_iii_db
    restart: always
    env_file: .db.env
    networks:
      - firefly_iii
    volumes:
      - firefly_iii_db:/var/lib/mysql
  cron:
    #
    # To make this work, set STATIC_CRON_TOKEN in your .env file or as an environment variable and replace REPLACEME below
    # The STATIC_CRON_TOKEN must be *exactly* 32 characters long
    #
    image: alpine
    restart: always
    container_name: firefly_iii_cron
    command: sh -c "echo \"0 3 * * * wget -qO- http://app:8080/api/v1/cron/Z%qn321j&Sppoi7zH&XI2iRCe%^3e4r^;echo\" | crontab - && crond -f -L /dev/stdout"
    networks:
      - firefly_iii

volumes:
   firefly_iii_upload:
   firefly_iii_db:

networks:
  firefly_iii:
    driver: bridge

**Comment by shibinbin8** [comment]
but port is open:
![image](https://github.com/user-attachments/assets/727754b0-7a7f-46d4-8632-85f316ac4c62)

**Comment by JC5** [comment]
It looks like you have already registered an account. Please use it to login.

**Reply by shibinbin8** [reply]
![image](https://github.com/user-attachments/assets/88216745-7a80-4c2a-a01c-c38910102189)
but I can't , show this message .
